source/code/plugins/filter_scom_cor_match.rb

```diff
@@ -1,5 +1,7 @@
+require_relative 'scom_common'
+
 module Fluent
-  class SCOMCorrelatedMatchFilter < Filter
+  class SCOMCorrelatedMatchFilter < SCOMTimerFilterPlugin
```

Comment:
@lagalbra 
If it is fine by you I would like to merge both parts together this time :) . Moving forward I will keep the ruby and non-ruby commits separate.